### ISSUE TITLE
After TDLib-Update to TDLib 1.7.9 only pinned Chats show

### ISSUE THREAD

**Issue by broncheolus** [Wunderfitz/harbour-fernschreiber#458]
Device: XA2
SFOS: 4.3.0.12
Fernschr.: 0.11

After the latest unbelievably quick update to TDLib 1.7.9 only the pinned Chats show. The other chats reappear when one uses the search function. When typing the name of the disappeared chat partner, it reappears in the chat overview. Nothing has to be found, when searching, just doing so does the trick. 
You have to enter one name after the other into the search field and all the chats reappear. It is not necessary to switch back to the overview after each search. Only once is enough.

Keep up your great work. Greetings from Austria, broncheolus.

Edit: Maybe it´s #457, so: Sorry for filing an extra issue.

**Comment by Wunderfitz** [comment]
One user had a similar issues yesterday evening. Could you please try to logoff (About page) and login again? Moreover, you could try removing ~/.local/share/harbour-fernschreiber and restart the app.

**Comment by broncheolus** [comment]
Thanks. Log off and on did the trick. One little thing: There just seems to be a fragment of a former message of a group that I converted to a supergroup. This message is seen as a separate chat although it´s part of this group chat. I hope you understand what I mean. ;-)
Thanks for your help.

**Comment by Wunderfitz** [comment]
Good to see that this worked.

Yes, I know of this fragment, but as it happens so rarely, I haven't implemented something to fix that. Feel free to open an issue if it bugs you, but I have to say that it wouldn't be on my priority list for now. So a contribution from somebody else might be needed.

I close this one and pin it as it seems to happen from time to time.

**Comment by broncheolus** [comment]
Hi Wunderfitz!
I´m sorry for reopening this issue. Your workaround only worked for the fresh start after logging off. When I closed the app and opened it again the issue was back again. Shall I post the output from commandline when starting the app?

**Comment by Wunderfitz** [comment]
Huh, really strange. Feel free to do so, but please follow the instructions at https://github.com/Wunderfitz/harbour-fernschreiber#debug and keep in mind that you remove personal data if required.

**Comment by broncheolus** [comment]
Hi Wunderfitz!

It´s me once again. I just found out how to fix this issue. Maybe it helps you fixing this problem without me posting the output from commandline. 
This issue also appeared on my second device after updatin Fernschreiber (Xperia 10).

Workaround: 
1) Deactivating offline-caching brings all chats automatically back at once without the workaround mentioned above (searching)
2) Reactivating offline-caching, restarting (not logging out from) Fernschreiber keeps chats visible.
3) Restarting Fernschreiber a second time with caching enabled, lets all the not pinned chats disappear again
4) Repeating 1) repeats the observed behavior of Fernschreiber

Good luck, I hope it helps. Broncheolus.

**Comment by Wunderfitz** [comment]
Really strange... Then it really seems to be something in TDLib if en/disabling the online-only mode makes a difference. Unfortunately, I can't do much about it at the moment. 
Are you using some special features of Telegram, e.g. folders, special chat lists?

**Comment by broncheolus** [comment]
I use groups and one pubic information channel paired with a channel to comment. That´s all.
... and please, no hurry. Deactivating offline caching helps for the moment.

**Comment by Wunderfitz** [comment]
@broncheolus Do you still face this issue with the newest release? We are now a few TDLib releases ahead of the one that obviously created this issue...

**Comment by broncheolus** [comment]
Yes, problem still persists. When changing settings to "offline mode" everything seems fine after restart. When restarting a second time only groups and one person shows. When entering one of these chats all the others also appear but the order is not chronological. After changing to online mode and restart everything including order is fine.
